### ISSUE TITLE
An error CS1519 is reported twice at the same location in presence of a block and an arrow expression clause.

### ISSUE THREAD

**Issue by AlekseyTs** [dotnet/roslyn#2052]
See unit-test BlockBodyAndExpressionBody_03 in src/Compilers/CSharp/Test/Semantic/Semantics/ExpressionBodiedMemberTests.cs

The following error is reported twice at the same location:
error CS1519: Invalid token ';' in class, struct, or interface member declaration

**Comment by jcouv** [comment]
Seems like `BlockBodyAndExpressionBody_03` was updated since the bug was filed.

Look at `AttributeInMethodBody` instead for repro:

```C#
        [Fact, WorkItem(529895, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/529895")]
        public void AttributeInMethodBody()
        {
            var test = @"
public class Class1 
{
    int Meth2 (int parm) {[Foo(5)]return 0;}
}
";
            ParseAndValidate(test,
                // (4,27): error CS1513: } expected
                Diagnostic(ErrorCode.ERR_RbraceExpected, "[").WithLocation(4, 27),
                // (4,35): error CS1519: Invalid token 'return' in class, struct, or interface member declaration
                Diagnostic(ErrorCode.ERR_InvalidMemberDecl, "return").WithArguments("return").WithLocation(4, 35),
                // (4,35): error CS1519: Invalid token 'return' in class, struct, or interface member declaration
                Diagnostic(ErrorCode.ERR_InvalidMemberDecl, "return").WithArguments("return").WithLocation(4, 35),
                // (5,1): error CS1022: Type or namespace definition, or end-of-file expected
                Diagnostic(ErrorCode.ERR_EOFExpected, "}").WithLocation(5, 1));
        }
```

**Comment by CyrusNajmabadi** [comment]
@jcouv it looks like AttributeInMethodBody was updated as well.  Can this be closed out?
